SecretAgent/SecretAgent/Screen/Board/View/BoardViewController.swift

```diff
@@ -0,0 +1,142 @@
+        }
+
+        // MARK: - 위의 코드는 모두 임시 코드입니다.
+
+        stackView.distribution = .fillEqually
+        stackView.axis = .horizontal
+        stackView.backgroundColor = .gray
+        return stackView
+    }()
+
+    private lazy var badgeCollectionFlowLayout: UICollectionViewFlowLayout = {
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .vertical
+        layout.minimumLineSpacing = BoardSize.collectionViewLineSpacing
+        layout.sectionInset = BoardSize.collectionViewInsets
+        layout.itemSize = .init(width: view.frame.size.width, height: BoardSize.badgeHeight)
+
+        return layout
+    }()
+
+    private lazy var badgeCollectionView: UICollectionView = {
+        let collectionView = UICollectionView(frame: view.frame, collectionViewLayout: badgeCollectionFlowLayout)
+        collectionView.register(cell: CustomCollectionViewCell.self, forCellReuseIdentifier: CustomCollectionViewCell.identifier)
+        return collectionView
+    }()
+
+    // MARK: - Life Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setDelegateAndDataSource()
+    }
+
+    override func render() {
+        view.addSubview(fixedBadgeInformation)
+        fixedBadgeInformation.snp.makeConstraints { make in
+            make.top.equalTo(view.safeAreaLayoutGuide)
+            make.leading.trailing.equalToSuperview()
+            make.height.equalTo(BoardSize.upperBadgeInfoHeight)
+        }
+
+        view.addSubview(badgeCollectionView)
+        badgeCollectionView.snp.makeConstraints { make in
+            make.top.equalTo(fixedBadgeInformation.snp.bottom)
+            make.leading.trailing.bottom.equalToSuperview()
+        }
+    }
+
+    override func configUI() {
+        view.backgroundColor = .systemBackground
+    }
+
+    // MARK: - Func
+
+    @objc func showDropdown() {
+        // TODO: - 드롭다운 보여주기
+    }
+
+    private func setDelegateAndDataSource() {
+        badgeCollectionView.delegate = self
+        badgeCollectionView.dataSource = self
+    }
+}
+
+// MARK: UICollectionViewDelegate, UICollectionViewDataSource
+
+extension BoardViewController: UICollectionViewDelegate, UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return 25
+    }
+
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let myCell = collectionView.dequeueReusableCell(withReuseIdentifier: CustomCollectionViewCell.identifier, for: indexPath) as? CustomCollectionViewCell
+
+        let activeCoin = ImageLiteral.activeCoin
+        let inactiveCoin = ImageLiteral.inactiveCoin
+
+        if indexPath.row < totalBadgeNumber {
+            myCell?.badgeImageView.image = activeCoin
+        } else {
+            myCell?.badgeImageView.image = inactiveCoin
+        }
```

Comment:
p5;
```suggestion
        if indexPath.row < totalBadgeNumber {
            myCell?.badgeImageView.image = ImageLiteral.activeCoin
        } else {
            myCell?.badgeImageView.image = ImageLiteral.inactiveCoin
        }
```
이렇게하면 코드 줄수가 더 짧아집니다!